extensions/amp-story-dev-tools/0.1/amp-story-dev-tools-tab-preview.css

```diff
@@ -0,0 +1,243 @@
+  display: flex !important;
+  flex-direction: column !important;
+  overflow: hidden !important;
+  position: absolute !important;
+  margin: auto !important;
+  height: fit-content !important;
+  top: 0 !important;
+  bottom: 0 !important;
+  padding: 10px 8px !important;
+  height: fit-content !important;
+  opacity: 0;
+  transition: opacity 0.15s ease-in-out;
+}
+
+.i-amphtml-story-dev-tools-device-screen {
+  border-radius: 30px !important;
+  background: #2E3137 !important;
+  overflow: hidden !important;
+  padding-top: 40px;
+}
+
+amp-story-player a {
+  display: none !important;
+}
+
+.i-amphtml-story-dev-tools-device-chips-container {
+  position: absolute !important;
+  bottom: 0 !important;
+  width: 100% !important;
+  text-align: center !important;
+}
+
+.i-amphtml-story-dev-tools-device-chip {
+  border-radius: 16px !important;
+  background: #2e2e2e !important;
+  border: 1px solid #2e2e2e !important;
+  height: 32px !important;
+  font-size: 13px !important;
+  line-height: 20px !important;
+  display: inline-flex !important;
+  align-items: center !important;
+  padding: 0 6px 0 10px !important;
+  margin: 0 5px !important;
+  max-width: 300px !important;
+  transition: background 0.2s ease-in-out, color 0.2s ease-in-out, max-width 0.2s ease-in-out !important;
+  white-space: nowrap !important;
+  cursor: pointer !important;
+}
+
+.i-amphtml-story-dev-tools-device-chip:hover {
+  background: #3d3d3d !important;
+}
+
+.i-amphtml-story-dev-tools-device-chip svg {
+  cursor: pointer !important;
+  background: black !important;
+  border-radius: 50% !important;
+  padding: 2px !important;
+  margin-left: 10px !important;
+  transform: rotate(0deg) !important;
+  transition: transform 0.2s ease-in-out !important;
+}
+
+.i-amphtml-story-dev-tools-device-chips .i-amphtml-story-dev-tools-device-chip[inactive] {
+  max-width: 0px !important;
+}
+
+.i-amphtml-story-dev-tools-device-chip[inactive] {
+  background: transparent !important;
+}
+
+.i-amphtml-story-dev-tools-device-chip[inactive]:hover {
+  background: #1d1d1d !important;
+}
+
+.i-amphtml-story-dev-tools-device-chip[inactive] svg {
+  transform: rotate(90deg) !important;
+  background: #ffffff22 !important;
+}
+
+.i-amphtml-story-dev-tools-device-chip:not([inactive]):not(.i-amphtml-story-dev-tools-add-device) .i-amphtml-story-dev-tools-device-chip-add-stick {
+  fill: transparent !important;
+}
+
+.i-amphtml-story-dev-tools-device-chip.i-amphtml-story-dev-tools-add-device {
+  background: #79B3FF !important;
+  color: black !important;
+}
+
+.i-amphtml-story-dev-tools-device-chip.i-amphtml-story-dev-tools-add-device svg {
+  fill: #79B3FF !important;
+  background: black !important;
+}
+
+.i-amphtml-story-dev-tools-device-chip.i-amphtml-story-dev-tools-add-device:hover {
+  background: #92c1ff !important;
+}
+
+.i-amphtml-story-dev-tools-device-popup-bg {
+  position: fixed !important;
```

Comment:
Given that all our UI is full page and we don't have any scroll, why use `fixed` over `absolute`? Maybe you need to change the `.i-amphtml-story-dev-tools-device-popup-bg` DOM element parent but sticking to `absolute` sounds more predictable.

Reply:
I planned to design this so that technically you could just make an `<amp-story-dev-tools-tab-preview>` element without needing the parent `<amp-story-dev-tools>`, so `fixed` helps make the background cover the whole screen instead of just the element. Maybe appending it to the root of the document is a good idea, WDYT? Not sure what the benefits are of fixed vs absolute

Reply:
Edit: appending the dialog to the parent / body is a bad idea since we want to handle event delegation on the component, so the dialogs should be attached to the preview tab for that to work. Fixed makes the most sense to me in this case, and we can keep the event delegation logic on just the `amp-story-dev-tools-tab-preview` element.